runtime/doc/options.txt

```diff
@@ -5268,8 +5268,8 @@ A jump table for the options with a short description can be found at |Q_op|.
 	The name of the temporary file can be represented by "%s" if necessary
```

Comment:
Does the defaults list on line 5269 need to be updated? Looks like `src/nvim/options.lua` only has these two:

```
'shellpipe' 'sp'	string	(default "| tee" or "2>&1| tee")
```

Reply:
I am confused as to how `options.lua` works (by which I mean I haven't looked into it at all).  It says, if non-windows, shellpipe will be "| tee", but in Linux with bash, I get shellpipe "2>&1| tee".  That's why I didn't know if I should change the default values in the doc.

What I _can_ say is we can remove the old cmd default: ">%s 2>&1".  But I don't know if we can remove ">", "|& tee".  Should I remove only the old-cmd default and keep the rest? or remove all but the two listed in `options.lua`?

Reply:
Oh I see we have special handling for 'shellpipe': https://github.com/neovim/neovim/blob/86c164d2a14382adfb388a66699828df6c623102/src/nvim/option.c#L629-L643

> Should I remove only the previous-cmd default and keep the rest?

Yes